# Representation of a vehicle state in the SML World

float64 x
float64 y

float64 yaw
